src/Nethermind/Nethermind.Mev/Source/BundlePool.cs

```diff
@@ -248,7 +248,7 @@ private bool ValidateBundle(MevBundle bundle)
                 return false;
             }
 
-            IReleaseSpec spec = _specProvider.GetSpec(bundle.BlockNumber);
+            IReleaseSpec spec = _specProvider.GetSpec(bundle.BlockNumber, (ulong)bundle.MinTimestamp);
```

Comment:
Really don't know, but maybe we should use the current time stamp? We can't go back to the past.